src/scripts/hamburger.js

```diff
@@ -0,0 +1,22 @@
+function burgerNavigation() {
+  const mobileIcon = document.getElementById("mobile-navigation");
+  const mobileBars = mobileIcon.querySelector(".nav-bars");
+  const navigation = document.getElementById("navigation-list");
```

Comment:
this is a list, not the navigation itself. Fix naming

Reply:
Changed